ruby/hamsters.rb

```diff
@@ -0,0 +1,134 @@
+puts "Name of hamster?"
```

Comment:
This file should be in a separate Pull Request.  Make sure you're using your branches for each feature individually.